### PR TITLE
tools: refine sync-diff-inspector's document in v2.1 and v3.0

### PR DESCRIPTION
**Pull request by WangXiangUSTC**
### What is changed, added or deleted? 
refine sync-diff-inspector's document in v2.1 and v3.0

### What is the related PR or file link(s)? 

already done in `dev` https://github.com/pingcap/docs-cn/pull/1637

### Which version does your change affect? 
v2.1 and v3.0

### CODE REVIEW

File: v3.0/reference/tools/sync-diff-inspector/overview.md [hunk 1]
```diff
@@ -0,0 +1,214 @@
+---
+title: sync-diff-inspector 用户文档
+category: tools
```
Comment (dcalvin) [inline v3.0/reference/tools/sync-diff-inspector/overview.md]:
```suggestion
category: tools
aliases: ['/reference/tools/sync-diff-inspector']
```

Comment (lilin90) [reply v3.0/reference/tools/sync-diff-inspector/overview.md]: @dcalvin For this `aliases`, it should be `['docs-cn/v3.0/reference/tools/sync-diff-inspector/']`.

File: v2.1/reference/tools/sync-diff-inspector/overview.md [hunk 2]
```diff
@@ -0,0 +1,214 @@
+---
+title: sync-diff-inspector 用户文档
+category: tools
```
Comment (lilin90) [inline v2.1/reference/tools/sync-diff-inspector/overview.md]:
```suggestion
category: tools
aliases: ['/docs-cn/v2.1/reference/tools/sync-diff-inspector/']
```

File: v3.0/reference/tools/sync-diff-inspector/overview.md (hunk 1 above)
Comment (WangXiangUSTC) [reply v3.0/reference/tools/sync-diff-inspector/overview.md]: added

**Comment by dcalvin** [comment]
LGTM

**Pull request merged**